R/pull.R

```diff
@@ -1,15 +1,17 @@
 #' @export
 #' @importFrom dplyr pull
-pull.tbl_graph <- function(.data, var = -1) {
+pull.tbl_graph <- function(.data, var = -1, name = NULL) {
   d_tmp <- as_tibble(.data)
   var <- enquo(var)
-  pull(d_tmp, !! var)
+  name <- enquo(name)
+  pull(d_tmp, !! var, !! name)
```

Comment:
might be replaced by `pull(d_tmp, {{var}}, {{name}})`

Reply:
yeah... this is fine. I haven't double-braced the tidy eval code in tidygraph so this fits the rest of the code